test.ts

```diff
@@ -129,7 +132,70 @@ Deno.test("main", async (t) => {
     await relay.shutdown();
 });
 
-Deno.test("graphql", async () => {});
+// https://github.com/nostr-protocol/nips/blob/master/11.md
+Deno.test("NIP-11: Relay Information Document", async (t) => {
+    try {
+        await Deno.remove("test.sqlite");
```

Comment:
maybe we can move this logic to `test_kv`

Reply:
ok